VideoWeb/VideoWeb/ClientApp/src/app/shared/self-test/self-test.component.ts

```diff
@@ -242,10 +242,12 @@ export class SelfTestComponent implements OnInit, OnDestroy {
             this.videoCallService.updateMicrophoneForCall(mic);
         }
         this.preferredMicrophoneStream = await this.userMediaStreamService.getStreamForMic(mic);
-        this.logger.info(`${this.loggerPrefix} Update camera and microphone selection`, {
-            cameraId: cam.deviceId,
-            microphoneId: mic.deviceId
-        });
+        if (cam && mic) {
```

Comment:
we should log the device for this stream, not only log when both mic and camera are set